### ISSUE TITLE
HR new faces/new places channel

### ISSUE THREAD

**Issue by kelslundberg** [betheluniversity/portal#43]
We talked about creating a data definition for new faces/new places channel to make formatting and loading easier for HR.

Fields:

- Name
- Title
- Office/Department
- Image (link to directory photo)

They want a different name for this channel too, which Erik and I will need to think of...

Also - @c-schwarze and @e-jameson - I don't know if you all have talked about this yet, but if web authors are editing content, will they have access to this Portal area of the CMS? I'm assuming all channel blocks need to be in the Portal folder...or was there ever talk of having content within someone's site folder that then feeds into the Portal? Is this something I should add to the meeting agenda?

**Comment by e-jameson** [comment]
We are going to brainstorm this tomorrow (May 5). If we come up with a solution that allows them to maintain the content in their site we'll do that I think. Otherwise we'll find a way for them to edit only HR content in the Portal folder.

**Comment by kelslundberg** [comment]
Sounds great!

**Comment by c-schwarze** [comment]
Phil created this (still needs more styling) on the homepage (bottom right)

**Comment by e-jameson** [comment]
Closing this for now. @kelslundberg if we decide we want to split HR News into more channels, lets create a new issue to do that.

**Comment by kelslundberg** [comment]
Need a fixed with for the directory photos :) Not sure if this is @c-schwarze or @johjacb 
![screen shot 2017-05-22 at 4 13 19 pm](https://cloud.githubusercontent.com/assets/6525391/26328693/b9e3a5e0-3f09-11e7-999f-f2b62a113a5c.png)

**Comment by e-jameson** [comment]
@phg49389 can you update the faces/places format to use the same #renderImage macro that the Events channel uses:

https://cms.bethel.edu/entity/open.act?id=6ae199fe8c5865135482d2e5538c5c31&type=format&

**Comment by johjacb** [comment]
should be updated. 

also I think leaving the image url is fine as is, because we actually don't want a big version at any size. so, should be good as it stands

**Comment by kelslundberg** [comment]
HR is really liking this and thinks it will be way easier for them!

Just a couple things from our meeting:

Is this set to pull a certain number? Michelle wants to be able to have it display however many she wants and she can manually deactivate when they should come doe.

Change the name to Fresh Starts and Sendoffs

Add drop down selection with these options so all they needs to do is select and these then show up where "New Face" is right now: 
Fresh Start
Send Off

**Comment by josiahtillman** [comment]
Right now it is just pulling the most recent 5, or at least it is supposed to. We could add a field on cascade that would allow you to choose which ones to display. Other than that, I don't think there is a way for us to deactivate them from portal. It has to be done on cascade's side.

I'll start making those other changes.

**Comment by e-jameson** [comment]
A couple thoughts. @c-schwarze what do you think about these ideas?

- we could turn it into a block chooser, and she can choose which ones to include or not include and can determine the order as well.

- We could pull the latest 10, but add an `activated`/`deactivated` radio button to each one and skip if its deactivated. 

The first option would make our code easier, but its a little less magic going on for her, but a tiny bit more upkeep.

**Comment by c-schwarze** [comment]
I think if she wants to fully control these, we should set it up so that the new faces and places shows all people who are activated and have her fully control it. We then need to add the Fresh Starts and Sendoffs dropdown to each one.

**Comment by kelslundberg** [comment]
@c-schwarze sure! so if I'm getting this right, she could activate or deactivate any block and then it would display all active blocks?

**Comment by c-schwarze** [comment]
Yep! It should update each night when we publish it, otherwise she could publish the page. There is a chance we could autopublish the page with a workflow whenever a new face/new place gets updated

Also, are we changing the name of this to fresh start and send offs? If so, we should commit now, so it matches up later

**Comment by kelslundberg** [comment]
Sorry for my delay! Sounds good on the updating. I just heard back from HR now that they see what "Fresh Starts" looks like in the channel. They've decided on:

Channel name: 
Employee Updates

Options: 
- New Employee
- New Role
- Send-Off

**Comment by tim-heck** [comment]
I've made these changes to the channel in cascade, should read 'Employee Updates' with those three options for labeling the employees.

**Comment by kelslundberg** [comment]
Thanks @tim-heck - looks great!

Hey, @d-freed - this is an area of the portal that Michelle in HR will be updating. It's the replacement for New Faces/New Places. Each employee is a block. She can create new blocks and activate the ones she wants to display in MyBethel. 

Do you want to take a look at how it's set up in Cascade and figure out where you want this to live and if you want any kind of workflow on the content?

https://cms.bethel.edu/entity/open.act?id=99d57e888c58651360b6175ff71b8e6f&type=folder&
